### PR TITLE
dbeaver/pro#1095 support for Double type added to Snowflake

### PR DESCRIPTION
**Pull request by LonwoLonwo**
2022-12-08 01_42_47-Schema Compare

### CODE REVIEW

File: src/main/java/liquibase/ext/snowflake/datatype/SnowflakeDoubleType.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import liquibase.datatype.core.DoubleType;
+import liquibase.ext.snowflake.database.SnowflakeDatabase;
+
+@DataTypeInfo(
+	    name = "double",
+	    aliases = {"java.sql.Types.DOUBLE", "java.lang.Double"},
+	    minParameters = 0,
+	    maxParameters = 2,
+	    priority = LiquibaseDataType.PRIORITY_DATABASE
+	)
```
Comment (E1izabeth) [inline src/main/java/liquibase/ext/snowflake/datatype/SnowflakeDoubleType.java:16]:
```suggestion
@DataTypeInfo(
    name = "double",
    aliases = {"java.sql.Types.DOUBLE", "java.lang.Double"},
    minParameters = 0,
    maxParameters = 2,
    priority = LiquibaseDataType.PRIORITY_DATABASE
)
```

File: src/main/java/liquibase/ext/snowflake/datatype/SnowflakeDoubleType.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+public class SnowflakeDoubleType extends DoubleType {
+	
+	public SnowflakeDoubleType() {
+		
+	}
+	
+	public int getPriority() {
+        return 5;
+    }
```
Comment (E1izabeth) [inline src/main/java/liquibase/ext/snowflake/datatype/SnowflakeDoubleType.java:25]: Please, fix indentation here too.

File: src/main/java/liquibase/ext/snowflake/datatype/SnowflakeDoubleType.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+	    priority = LiquibaseDataType.PRIORITY_DATABASE
+	)
+public class SnowflakeDoubleType extends DoubleType {
+	
+	public SnowflakeDoubleType() {
+		
+	}
+	
+	public int getPriority() {
+        return 5;
```
Comment (E1izabeth) [inline src/main/java/liquibase/ext/snowflake/datatype/SnowflakeDoubleType.java:24]: Why 5? What is 5?

**Review by E1izabeth — CHANGES_REQUESTED** [review]

File: src/main/java/liquibase/ext/snowflake/datatype/SnowflakeDoubleType.java (hunk 3 above)
Comment (LonwoLonwo) [reply src/main/java/liquibase/ext/snowflake/datatype/SnowflakeDoubleType.java]: LiquibaseDataType.PRIORITY_DATABASE
Standard of Liquibase

**Review by E1izabeth — APPROVED** [review]

**Pull request merged**